robot-db/src/main/java/domain/Author.java

```diff
@@ -0,0 +1,36 @@
+package domain;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import java.util.Collection;
+
+/**
+ * Author representation in Database
```

Comment:
English: either a Database or the Database, but it can't be just "Database". They really notice this stuff.